Examples/UsingDictionaryExample.py

```diff
@@ -74,16 +72,16 @@ def artistImageGetter(track):
                 return groupImage
 
         self.myOlv.SetColumns([
-            ColumnDefn("Title", "left", -1, "title", imageGetter=musicImage),
-            ColumnDefn("Artist", "left", -1, "artist", imageGetter=artistImageGetter),
-            ColumnDefn("Album", "center", -1, "album")
+            OLV.ColumnDefn("Title", "left", -1, "title", imageGetter=musicImage),
+            OLV.ColumnDefn("Artist", "left", -1, "artist", imageGetter=artistImageGetter),
+            OLV.ColumnDefn("Album", "center", -1, "album")
         ])
         self.myOlv.SetObjects(self.listOfDictionaries)
 
 if __name__ == '__main__':
-    app = wx.PySimpleApp(0)
+    print('Using {} ({}) from {}.'.format(OLV.__name__, OLV.__version__, OLV.__path__))
```

Comment:
You should use an f-string here

Reply:
You are absolutely right here. I copied this line from the other examples without any changes.

There are many other strings, which ask for f-strings. I do suggest that we tackle all these in one PR. I can do it, if you want.